src/Symfony/Component/Message/composer.json

```diff
@@ -0,0 +1,39 @@
+{
+    "name": "symfony/message",
+    "type": "library",
+    "description": "Symfony Message Component",
+    "keywords": [],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Samuel Roze",
+            "email": "samuel.roze@gmail.com"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": "^7.1.3"
+    },
+    "require-dev": {
+        "symfony/serializer": "~3.4|~4.0"
+    },
+    "suggest": {
+        "sroze/enqueue-bridge": "For using the php-enqueue library as an adapater."
```

Comment:
typo adapater -> adapter

Reply:
Thank you, Pascal. Yet another typo found 😄Fixed in 63a64ea7ffd2e2696f7753b00542e13d4bba792d.